pkg/batchscheduler/volcano/volcano_scheduler.go

```diff
@@ -115,9 +115,12 @@ func (v *VolcanoBatchScheduler) getAppPodGroupName(app *v1beta2.SparkApplication
 }
 
 func (v *VolcanoBatchScheduler) syncPodGroup(app *v1beta2.SparkApplication, size int32, minResource corev1.ResourceList) error {
-	var err error
+	var (
+		err error
+		pg  *v1beta1.PodGroup
+	)
```

Comment:
This change doesn't seem necessary given that `pg` is not used outside the if-else scope.

Reply:
You are right. I removed this change.

Reply:
what do you prefer ?